docs/cookbook/platform-sh.rst

```diff
@@ -201,8 +201,14 @@ Remember to have it imported in the config:
 
     # app/config/config.yml
     imports:
+        # - { resource: parameters.yml } <- Has to be place before our new file
```

Comment:
why do you have this line commented?
`Has to be place...` => `Has to be placed...`